tests/dark_storage/test_http_endpoints.py

```diff
@@ -40,3 +41,14 @@ def test_get_experiment_ensemble(poly_example_tmp_dir, dark_storage_client):
     assert len(ensembles_json) == 1
     assert ensembles_json[0]["experiment_id"] == experiment_json[0]["id"]
     assert ensembles_json[0]["userdata"]["name"] == "default"
+
+
+def test_get_ensemble_parameters(poly_example_tmp_dir, dark_storage_client):
+    ensemble_id = uuid.uuid4()
```

Comment:
sorry, did not discover this when i looked at the code earlier.. but would it make more sense doing something like 
```
resp: Response = dark_storage_client.get("/experiments")
    experiment_json = resp.json()
    ensemble_id = experiment_json[0]["ensemble_ids"][0]
```
to get the ensemble id to ask for instead of generating an id? And with that it should be possible to avoid the changes in lines 56-61 in `enkf.py` - `get_name()`?

Reply:
The best way of solving the changes in `enkf.py` is probably to update the `ids` during initialization/startup. This way calling GET /experiments/some random uuis/parameters would not crash

Reply:
guess this is a perfect issue to discuss on tuesday on our small technical planning - how to initialize and generate the id's... ?